app/javascript/channels/consumer.js

```diff
@@ -3,4 +3,7 @@
 
 import { createConsumer } from "@rails/actioncable"
 
-export default createConsumer()
+const App = {};
+App.cable = createConsumer();
+
+export default App;
```

Comment:
I'm not sure what the point of this `App` object is, but if I remove it everything seems to break. Before the upgrade to Rails 6/Webpack, this looked a lot different:

https://github.com/vindennl48/HipBox/blob/d6d1f128d7ca808884a542dc94f0e8594b51bf41/app/assets/javascripts/cable.js#L8-L13

Do you see anything concerning about my implementation?

I think this might be an example of where we're fighting against the Rails way of doing ActionCable - maybe the solution here is just to refactor this code to be more Railsy?

Reply:
So, this code is kind of the foundation for actioncable, and its actually not my code it was auto-generated for actioncable.  Be it that it was generated by them, I think its as railsy as it gets lol.

Reply:
Fair enough, I worded my question to you terribly... let's revisit this in-person.